streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamJoinTest.java

```diff
@@ -57,7 +57,7 @@
 import org.apache.kafka.test.GenericInMemoryKeyValueStore;
 import org.apache.kafka.test.MockApiProcessor;
 import org.apache.kafka.test.MockApiProcessorSupplier;
-import org.apache.kafka.test.MockInternalNewProcessorContext;
+import org.apache.kafka.test.MockInternalProcessorContext;
```

Comment:
Follow up clean from https://github.com/apache/kafka/pull/18103

Rename this class, what result in lot of updates in many files...